charts/tractusx-connector-memory/README.md

```diff
@@ -41,7 +41,7 @@ Note that `DAPS_CERT` contains the x509 certificate, `DAPS_KEY` contains the pri
 
 ## Source Code
 
-* <https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector-memory>
+- <https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector-memory>
```

Comment:
Please move this change in the `README.md.gotmpl` file and regenerate the `README.md`

Reply:
don't think there is a way to do this in the `*gotmpl`. #359 excludes the `charts/` folder from the markdown linter, since we can only influence its generation in a very limited capacity.

Reply:
@florianrusch-zf I think this can stay as is now, with next generation it will be overridden anyway and Linter wouldn't complain anymore.